### PR TITLE
Add tags for Core Visual Graph

### PR DESCRIPTION
**Pull request by aldebaran30701**
Add tags for Core Visual Graph

### CODE REVIEW

File: CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/AddCustomBlazePlugin.java [hunk 1]
```diff
@@ -50,6 +52,7 @@
  */
 @ServiceProvider(service = Plugin.class)
 @NbBundle.Messages("AddCustomBlazePlugin=Add Custom Blazes")
+@PluginInfo(pluginType = PluginType.CREATE, tags = {"CREATE"})
```
Comment (antares1470) [inline CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/AddCustomBlazePlugin.java:55]: tag could possibly be MODIFY (similar to what you've done with RemoveBlazes). Else the RemoveBlazes tag should probably be DELETE

**Review by antares1470 — CHANGES_REQUESTED** [review]

File: CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/AddCustomBlazePlugin.java (hunk 1 above)
Comment (aldebaran30701) [reply CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/AddCustomBlazePlugin.java:55]: Good point, I think modify works best here.

**Review by antares1470 — APPROVED** [review]

File: CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/RemoveBlazePlugin.java [hunk 2]
```diff
@@ -45,6 +47,7 @@
  */
 @ServiceProvider(service = Plugin.class)
 @NbBundle.Messages("RemoveBlazePlugin=Remove Blazes")
+@PluginInfo(pluginType = PluginType.DELETE, tags = {"MODIFY"})
```
Comment (arcturus2) [inline CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/RemoveBlazePlugin.java:50]: what about `DELETE`? `MODIFY` also works but was curious to the thinking behind using `MODIFY`

File: CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/select/SelectAllPlugin.java [hunk 3]
```diff
@@ -37,7 +37,7 @@
  * @author algol
  */
 @ServiceProvider(service = Plugin.class)
-@PluginInfo(pluginType = PluginType.SELECTION, tags = {"SELECTION"})
```
Comment (arcturus2) [inline CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/select/SelectAllPlugin.java:40]: :clap:

**Review by arcturus2 — APPROVED** [review]

File: CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/RemoveBlazePlugin.java (hunk 2 above)
Comment (aldebaran30701) [reply CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/RemoveBlazePlugin.java:50]: I think of delete as removing elements from the graph, not specifically attributes. I can change it to delete to be more consistent?

**Pull request merged**